pkg/log/log.go

```diff
@@ -0,0 +1,20 @@
+package log
+
+import (
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+func ConfigureGlobalLogger(logFilename string) {
```

Comment:
Splitting this out is a good addition, even outside of the need to because of the second subcommand.